solutions/instances/DA-types.md

```diff
@@ -88,3 +88,44 @@ cloud_logs_policies = [
   }
 ]
 ```
+
+## Metrics Router Routes <a name="metrics_router_routes"></a>
+
+The `metrics_router_routes` input variable allows you to provide a list of route that will be configured in the IBM Cloud Metrics Routing. Refer [here](https://cloud.ibm.com/docs/metrics-router?topic=metrics-router-about) for more information.
+
+- Variable name: `metrics_router_routes`.
+- Type: A list of objects. Each object represents a route.
+- Default value: An empty list (`[]`).
+
+### Options for metrics_router_routes
+
+  - `name` (required):  The name of the route.
+  - `rules` (required): The routing rules that will be evaluated in their order of the array.
+    - `action` (optional): The action if the inclusion_filters matches, default is send action. Allowed values are `send` and `drop`. You can configure up to 10 rules per route.
```

Comment:
`You can configure up to 10 rules per route.` --> If we are writing this one, it should be for `rules` not for `action`?